test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -1,6 +1,8 @@
 import os
 import re
 
+from packaging import version
```

Comment:
Since `version.parse` is the only function used from this module, you can change this to
```
from packaging.version import parse
```
or
```
from packaging.version import Version
```

Reply:
Sure, using Version bc it makes more semantic sense.